src/Text/Blaze/Svg/Internal.hs

```diff
@@ -2,8 +2,6 @@
 module Text.Blaze.Svg.Internal where
 
 import Control.Monad.State
-import Data.Monoid (mappend, mempty)
```

Comment:
We cannot just remove this if we want to still support `base < 4.8` (which we do).  Instead we should probably add a dependency on `base-compat` and import `Prelude.Compat`.  See https://github.com/haskell-compat/base-compat .